package.json

```diff
@@ -191,6 +193,7 @@
     "parse5-html-rewriting-stream": "6.0.1",
     "pidtree": "^0.5.0",
     "pidusage": "^2.0.17",
+    "piscina": "3.1.0",
```

Comment:
Very nice alternative to jest considering we no longer need to fallback using forked processes.